eventmesh-runtime/conf/eventmesh.properties

```diff
@@ -72,7 +72,7 @@ eventMesh.server.blacklist.ipv4=0.0.0.0/8,127.0.0.0/8,169.254.0.0/16,255.255.255
 eventMesh.server.blacklist.ipv6=::/128,::1/128,ff00::/8
 
 #connector plugin
-eventMesh.connector.plugin.type=standalone
+eventMesh.connector.plugin.type=rocketmq
```

Comment:
Use standalone

Reply:
Thank you for reviewing! Yes, you may notice that these config changes come from these commits:

![image](https://github.com/apache/eventmesh/assets/41445332/d1fbcc75-9902-477a-839b-22e00b1aad59)

Because this PR is still a draft and WIP, so these commits haven't been reverted.

Don't worry, I won't forget it.